Native Installation.md

```diff
@@ -0,0 +1,48 @@
+# Native Installation for Development
```

Comment:
In the future we could consider merging this chapter with `Developer Guide.md`.

Reply:
Yeah. But indeed I'd like to do that after the initial merge.

Reply:
- Merge Native Installation into Developer Guide?
- Add python 2.7 as req

Reply:
Separate npm installation for regular users